engine/samples/scene/main.cpp

```diff
@@ -13,18 +13,18 @@
 #include <cubos/engine/data/scene.hpp>
 #include <cubos/engine/ecs/camera.hpp>
 #include <cubos/engine/ecs/grid.hpp>
-#include <cubos/engine/ecs/local_to_world.hpp>
-#include <cubos/engine/ecs/position.hpp>
-#include <cubos/engine/ecs/rotation.hpp>
-#include <cubos/engine/ecs/scale.hpp>
+#include <cubos/engine/env_settings/plugin.hpp>
+#include <cubos/engine/file_settings/plugin.hpp>
 #include <cubos/engine/gl/frame.hpp>
-#include <cubos/engine/plugins/env_settings.hpp>
-#include <cubos/engine/plugins/file_settings.hpp>
-#include <cubos/engine/plugins/imgui.hpp>
-#include <cubos/engine/plugins/renderer.hpp>
-#include <cubos/engine/plugins/tools/settings_inspector.hpp>
-#include <cubos/engine/plugins/transform.hpp>
-#include <cubos/engine/plugins/window.hpp>
+#include <cubos/engine/imgui/plugin.hpp>
+#include <cubos/engine/renderer/plugin.hpp>
+#include <cubos/engine/tools/settings_inspector/plugin.hpp>
+#include <cubos/engine/transform/local_to_world.hpp>
+#include <cubos/engine/transform/plugin.hpp>
+#include <cubos/engine/transform/position.hpp>
+#include <cubos/engine/transform/rotation.hpp>
+#include <cubos/engine/transform/scale.hpp>
```

Comment:
Having to write this basically every time you use the plugin is a nuissance, I think we should either include those headers inside `transform/plugin.hpp` or define the components in that file instead of on separate headers.

Reply:
I'll move the headers to transform/plugin.hpp